tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -799,12 +799,13 @@ static void AddColumnsFromField(std::vector<RColumnMergeInfo> &columns, const RN
    // NOTE: here we can match the src and dst columns by column index because we forbid merging fields with
    // different column representations.
    for (auto i = 0u; i < srcFieldDesc.GetLogicalColumnIds().size(); ++i) {
+      if (srcFieldDesc.IsProjectedField())
+         continue;
+
```

Comment:
The commit message says to "add some extra verbose messages", but this is clearly a functional change (and should probably have a unit test, not just an integration test?)

Reply:
Yes, I split the changes in multiple commits after the fact and this must have slipped into the wrong one. I can put it into the proper commit and add a unit test for it